woocommerce/woocommerce
# Update pnpm once again to fix issues with auto-install-peers

### Changes proposed in this Pull Request:

There were issues with installing dependencies on a monorepo with auto-install-peers enabled. It would crash every time you try and install dependencies and this was causing some serious interruption to dev work.

This fixes that problem by updating the pnpm version constraint (once again) so that users can only use pnpm 7.13.3 and above (where this fix was released: https://github.com/pnpm/pnpm/pull/5467)

### How to test the changes in this Pull Request:

1. CI should pass
2. Locally you should be able to update your pnpm version `npm i -g pnpm` and run `pnpm i` with success. (make sure to clear the pnpm cache just in case: `pnpm store prune`

### FOR PR REVIEWER ONLY:
